ghcide/src/Development/IDE/Core/Rules.hs

```diff
@@ -182,6 +183,14 @@ instance Pretty Log where
     LogLoadingHieFileSuccess path ->
       "SUCCEEDED LOADING HIE FILE FOR" <+> pretty path
     LogExactPrint log -> pretty log
+    LogTypecheckedFOI path -> vcat
+      [ "WARNING: Typechecked a file which is not currently open in the editor:" <+> pretty (fromNormalizedFilePath path)
```

Comment:
Logging a message at INFO that begins with `WARNING` makes no sense. Use WARN!

Reply:
How does WARN manifest in the vscode UI?

Reply:
It appears in the logs, and I think it currently gets shown to the user. But I'm pretty convinced we should turn that off, and I would _much_ rather do that than start weirdly misclassifying logs in order to get different behaviour.

Reply:
Oh no, only errors get shown to the user at the moment. So it just goes into the log.